backend/RestAPI/Startup.cs

```diff
@@ -39,7 +39,9 @@ public void ConfigureServices(IServiceCollection services)
                 options.AddPolicy("CorsPolicy",
                     builder => builder.AllowAnyOrigin()
                         .AllowAnyMethod()
-                        .AllowAnyHeader());
+                        .AllowAnyHeader()
+                        .WithExposedHeaders("Content-Disposition"));
```

Comment:
Can you explain this change?

Reply:
It's so that frontend could see the filename that is being sent